Applications/spire/include/spire/book_view_ui_tester/random_book_view_model.hpp

```diff
@@ -99,8 +102,10 @@ namespace Spire {
         m_loading_flag;
       int m_quote_count;
 
+      Nexus::MarketCode get_random_market();
```

Comment:
This should be const.

Reply:
It was this one here, the m_random_engine() call isn't const.